docs/decisions/0005-PII-leakage-prevention.rst

```diff
@@ -0,0 +1,27 @@
+PII leakage prevention
+================================
+
+Status
+------
+
+Pending
+
+Context
+-------
+
+``Event-routing-backends`` transforms and emits edx events that may contain PII which is not meant to be shared with learning record consumers. New xAPI and Caliper transformers are expected to be added in ``Event-routing-backends`` and therefore, a mechanism needs to be put in place to reduce chances of PII leakage via these transformers.
+
+Decision
+--------
+#. An accessor method will be developed to get values from open edx events for a specified key. Information inside the open edx event will only be accessed using this accessor method in base transformers and in event transformers.
```

Comment:
[strengthen] Can you add a link to the accessor method so developers know which method is required to be used for PII protection?

Reply:
Sure.